test/ExecutionTests.jl

```diff
@@ -26,7 +25,9 @@ for s in sizes
 end
 
 groups["special"]["macro"] = @benchmarkable @test(1 == 1)
-groups["special"]["kwargs"] = @benchmarkable svds(rand(2, 2), nsv = 1)
+if "Arpack" in keys(Pkg.installed())
```

Comment:
I think we should just change the code being benchmarked to not use `svds`.

Reply:
That sounds good to me.